core/scripts/keystone/src/02_deploy_keystone_workflows.go

```diff
@@ -0,0 +1,173 @@
+package src
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	helpers "github.com/smartcontractkit/chainlink/core/scripts/common"
+	kcr "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+)
+
+type deployKeystoneWorkflows struct{}
+
+func NewDeployKeystoneWorkflowsCommand() *deployKeystoneWorkflows {
+	return &deployKeystoneWorkflows{}
+}
+
+func (g *deployKeystoneWorkflows) Name() string {
+	return "deploy-keystone-workflows"
+}
+
+func (g *deployKeystoneWorkflows) Run(args []string) {
+	fs := flag.NewFlagSet(g.Name(), flag.ContinueOnError)
```

Comment:
I was unsure if I should keep these sub-entrypoints around, or just remove them all and add toggling flags to `01_provision_keystone.go`. It will shrink the diff quite significantly if I did that